Presentation.Web/app/components/global-admin/global-admin-local-admins-root.component.ts

```diff
@@ -0,0 +1,158 @@
+        newOrg: Models.ViewModel.Select2.ISelect2OrganizationOptionWithCvrViewModel;
+
+        static $inject: string[] = ["userService", "organizationRightService", "select2LoadingService", "$scope", "notify"];
+        constructor(
+            private readonly userService: Services.IUserService,
+            private readonly organizationRightService: Services.Organization.IOrganizationRightService,
+            private readonly select2LoadingService: Services.ISelect2LoadingService,
+            private readonly $scope: ng.IScope,
+            private readonly notify) {
+        }
+        
+        $onInit() {
+            if (this.currentOrganizationId === null) {
+                throw "Missing parameter 'currentOrganizationId'";
+            }
+            if (this.userId === null) {
+                throw "Missing parameter 'userId'";
+            }
+
+            this.callbacks = {
+                removeRight: (rightId: number) => this.deleteLocalAdmin(rightId)
+            } as IGlobalAdminLocalAdminCallbacks;
+
+            this.loadData();
+        }
+
+        newLocalAdmin() {
+            // select2 changes the value twice, first with invalid values
+            // so ignore invalid values
+            if (typeof this.newUser !== "object") return;
+            if (!(this.newOrg && this.newUser)) return;
+
+            const userToUpdateId = this.newUser.id;
+            const newOrgId = this.newOrg.id;
+            var userName = this.newUser.text;
+            var orgName = this.newOrg.text;
+
+            const roleId = API.Models.OrganizationRole.LocalAdmin;
+
+            if (!(userToUpdateId && newOrgId && roleId)) return;
+            
+            var msg = this.notify.addInfoMessage("Arbejder ...", false);
+            const self = this;
+            this.organizationRightService.create(newOrgId, userToUpdateId, roleId)
+                .then(() => {
+                    msg.toSuccessMessage(userName + " er blevet lokal administrator for " + orgName);
+                    if (self.userId === userToUpdateId) {
+                        // Reload user
+                        self.userService.reAuthorize();
+                    }
+
+                    self.loadData();
+                }, () => {
+                    msg.toErrorMessage("Kunne ikke gøre " + userName + " til lokal administrator for " + orgName);
+                });
+
+            this.newOrg = null;
+            this.newUser = null;
+        }
+
+        deleteLocalAdmin(rightId: number) {
+            const localAdminRow = _.find(this.localAdmins, { id: rightId });
+            if (!localAdminRow)
+                return null;
+
+            const right = localAdminRow.objectContext;
+
+            const organizationId = right.organizationId;
+            const roleId = right.role;
+            var userId = right.userId;
+
+            return this.organizationRightService.remove(this.currentOrganizationId, organizationId, roleId, userId)
```

Comment:
this looks very wierd.. in the service method the first param is "roleOrg" but you are sending the current org.. I think you need to need to kill the `currentOrganizationId` and in the controller just reuse the "roleOrg" param when using it for the marker query param that the "currentOrgId" was used for